prompt2model/dataset_generator/prompt_based.py

```diff
@@ -442,7 +402,7 @@ def create_all_examples_dataset_and_generated_dataset(
     def compute_batch_size(
         self, expected_num_examples: int, generated_dataset: Dataset
     ) -> int:
-        """Computes the batch size for OpenAI API calls in a batch.
+        """Computes the batch size for API calls in a batch.
```

Comment:
Also, do these models support changing the `n` parameters?

Reply:
Seems so: https://litellm.readthedocs.io/en/latest/input/